tests/e2e-pw/specs/merchant/merchant-admin-disputes.spec.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe( 'Merchant disputes', () => {
+	// Use the merchant user for this test suite.
+	useMerchant();
+
+	test( 'Load the disputes list page', async ( { page } ) => {
+		await page.goto(
+			'/wp-admin/admin.php?page=wc-admin&path=/payments/disputes'
+		);
+
+		// Wait for the disputes table to load.
+		await page
+			.locator( '.woocommerce-table__table.is-loading' )
+			.waitFor( { state: 'hidden' } );
+
+		// .nth( 1 ) defines the second instance of the Disputes heading, which is in the table.
+		expect(
+			page
+				.getByRole( 'heading', {
+					name: 'Disputes',
+				} )
+				.nth( 1 )
+		).toBeVisible();
```

Comment:
Nitpick: This part is a bit hard to read. One way to improve that would be to remove the extra comma after "Disputes", so it can be formatted in a single line.

```suggestion
		expect(
			page.getByRole( 'heading', { name: 'Disputes' } ).nth( 1 )
		).toBeVisible();
```

Reply:
Thank you for the suggestion, applied in https://github.com/Automattic/woocommerce-payments/pull/10123/commits/3d48cdf8dfea693eaf2b271b7a436eef0b7c0da1